test/models/host_status/build_status_test.rb

```diff
@@ -15,7 +15,7 @@ def setup
   [ [true,  false, 'Pending installation'],
     [true,  true,  'Token expired'],
     [false, true,  'Installed'],
-    [false, false, 'Installed'] ].each do | waiting_for_build, token_expired, expectation |
+    [false, false, 'Installed'] ].each do |waiting_for_build, token_expired, expectation|
```

Comment:
Not sure why the hound is barking here, most like different defaults than clean rubocop. Locally I get no cop failures so I suggest ignoring this comment.

Reply:
personally I'd like to see these things like that:
```ruby
[
  [true,  false, 'Pending installation'],
  [true,  true,  'Token expired'],
  [false, true,  'Installed'],
  [false, false, 'Installed']
].each do |waiting_for_build, token_expired, expectation|
```
but no idea, how ruby-esque this would be.